### PR TITLE
Update runner guidelines.

### PR DESCRIPTION
**Pull request by ben-eb**
I've rewritten the guidelines to use a clear, more consistent tone. :smile:

### CODE REVIEW

File: Runner Guidelines.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ See also ClojureWerkz’s [open source project recommendations].
 
 ### 1.1. Accept functions in plugin parameters
 
-Some plugins accept function in parameters. If your runner has config file,
-it should be a Node.js module, because JSON format doesn’t support functions.
+PostCSS runners which use config files must be written in JavaScript, so that
+they can support plugins which accept a function, such as [`postcss-assets`]:
```
Comment (lydell) [inline Runner Guidelines.md]: It is no longer clear that the _config_ files should be written in JS. It sounds more as if the runners should be.

File: Runner Guidelines.md [hunk 2]
```diff
@@ -31,24 +31,26 @@ module.exports = [
...
 ```
 
-### 2.2. Use only asynchronous API
+### 2.2. Use only the asynchronous API
 
-PostCSS has synchronous API only for debug. Synchronous API is slower
-and can’t work with asynchronous plugins.
+PostCSS runners must use only the asynchronous API. The synchronous API is
+provided only for debugging, is slower, and can't work with asynchronous
+plugins:
```
Comment (lydell) [inline Runner Guidelines.md]: I think it was better without the colon here. Now it looks like it is going to be demonstrated why the sync API is discouraged.

**Comment by lydell** [comment]
Nice work!

**Comment by ben-eb** [comment]
Good points, thanks. Will update. :+1:

**Pull request merged**